spec/lib/tasks/create_provider_partnerships_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rake'
+
+describe 'creates providers partnerships from enrichments' do
+  subject do
+    Rake::Task['provider_partnerships:create_from_enrichments'].invoke
+  end
+
+  let!(:accredited_provider) { create(:accredited_provider) }
+  let!(:description) { 'The Description' }
+  let!(:training_provider) { create(:provider, accrediting_provider_enrichments: [AccreditingProviderEnrichment.new(UcasProviderCode: accredited_provider.provider_code, Description: description)]) }
+
+  Rails.application.load_tasks if Rake::Task.tasks.empty?
+
+  it 'calls Courses::PublishService service' do
```

Comment:
Could we surface the expected behaviour in the spec description here, please?

```suggestion
  it 'creates accredited partnerships for training partners from their enrichments' do
```